init web

Caiguer opened the pull request.

Commit:
init web

Commit:
fix hibernate

Commit:
hw12-web реализация web на jetty

stvort commented on HW12-WEB/src/main/java/com/oneso/web/helpers/HibernateHelper.java:
```diff
@@ -0,0 +1,42 @@
+
+  private static final Logger logger = LoggerFactory.getLogger(HibernateHelper.class);
+
+  public static ServiceUser createServiceUser() {
+    SessionFactory sessionFactory = HibernateUtils.buildSessionFactory("hibernate.cfg.xml",
+        User.class, Address.class, Phone.class);
+
+    SessionManagerHibernate sessionManager = new SessionManagerHibernate(sessionFactory);
+    UserDao userDao = new UserDaoHibernate(sessionManager);
+    ServiceUser serviceUser = new ServiceUserImpl(userDao, true);
+    serviceUser.saveUser(createUser("John", "+7 123-456-00", "#123"));
+
+    return serviceUser;
+  }
```
Лучше все же создать все снаружи, в `main`, а для инициализации БД начальными данными сделать отдельный сервис и вызвать его там же, в `main`, При таком подходе будет легко перейти на DI

stvort commented on HW12-WEB/src/main/java/com/oneso/web/helpers/HibernateHelper.java:
```diff
@@ -0,0 +1,42 @@
+    return serviceUser;
+  }
+
+  public static User createUser(String name, String phone, String address) {
```
Или можно сделать такой конструктор)

Caiguer committed:
изменения hibernate
теперь инициализация hibernate производится в main
для создания нового пользователя, теперь отдельный сервис
